lib/board_selection.py

```diff
@@ -0,0 +1,12 @@
+from .tictactoe_board import TicTacToeBoard
+
+
+class BoardSelection:
+    def __init__(self, ui):
+        self.ui = ui
+
+    def select_board_size(self):
+        self.ui.display("Welcome to TIC TAC TOE!")
+        self.ui.display("First, select a board size?")
+        size = int(self.ui.get_user_input())
+        return TicTacToeBoard(self.ui, size)
```

Comment:
There should be some input validation here that restricts the user to be able to only initialize 3x3 and 4x4 boards.

Reply:
Done!